### ISSUE TITLE
Postman - Unable to expand the left slider very far to the right

### ISSUE THREAD

**Issue by twallace0123** [a85/POSTMan-Chrome-Extension#10566]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

We have a bit longer Collection API call names but the slider does not expand any further like it used to.

### Steps To Reproduce

1. Open Postman
2.  Click on Collection
3.  Attempt to move the slider to the right
Actual: It only goes a small distance to the right

Expected: You should be able to resize it to whatever you desired or within reason at least.

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/52800145/149378256-e5422a5b-379e-4d0a-abdb-eac115544dba.png)

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Native App
- Postman Version: 9.82
```

**Comment by himanshu-webearl** [comment]
Is it resolved yet? As I m also facing this issue.

**Comment by vkaegis** [comment]
@twallace0123 Thanks for reporting this issue. This is fixed as of v10.8.8. Let me know if you run into any more issues around the resizing of the sidebar.
